odoo/tools/xml_utils.py

```diff
@@ -84,4 +84,6 @@ def create_xml_node(parent_node, node_name, node_value=None):
     :param str node_value: value of the created node (optional)
     :rtype: etree._Element
     """
-    return create_xml_node_chain(parent_node, [node_name], node_value)[0]
```

Comment:
i think this function is dead code now

Reply:
Both `create_xml_node_chain` and `create_xml_node` seem to be used a lot in SEPA.

Reply:
ah ok, my bad, didn't check enterprise modules